workspaces/ui-v2/src/contexts/config/AppConfiguration.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { ReactNode, useContext } from 'react';
 import invariant from 'invariant';
-interface IFeatureFlags {}
+interface IFeatureFlags {
+  showDeleteEndpointUi: boolean;
+}
 
```

Comment:
I know this interface was already here, but as we discussed after #681, we don't want to codify temporary feature flags (e.g. flags used for hiding features that will at some point be enabled and then no longer be configurable) as Typescript types. Instead, we'll want to use the `REACT_APP__` feature flags as described in [CreateReactApp](https://create-react-app.dev/docs/adding-custom-environment-variables). Yes, those are ugly, and get unwieldy over time, but all of that should be a driving force in getting rid of these feature flags as soon as possible, or getting rid features we'll never enable.